client/src/components/ui/pagination.tsx

```diff
@@ -0,0 +1,58 @@
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+
+type PaginationProps = {
+  currentPage: number;
+  totalPages: number;
+  onPageChange: (page: number) => void;
+}
+
+function PaginationArrow({ isDisabled, onClick, children, ...props }: { isDisabled: boolean; onClick: () => void; children: React.ReactNode }) {
```

Comment:
🌴 🌴 🌴 Non-blocking comment: Breaking out the props interface would make this a bit more readable, and follow idiomatic react typescript pattern.

```suggestion
interface PaginationArrowProps {
  isDisabled: boolean;  
  onClick: () => void;
  children: React.ReactNode;
}

function PaginationArrow({ isDisabled, onClick, children, ...props }: PaginationArrowProps) {
```